client/app/components/Estimation.js

```diff
@@ -19,79 +19,84 @@ import PopUp from './PopUp';
+  if (revealed) {
+    const estimations = selectedStory.get('estimations');
+    var storyPointConsentAchieved = isConsentAchieved(estimations);
+  }
+
+  return (
+    <div className="estimation">
+
+      <div className="selected-story">
+        <h4>
+          {selectedStory.get('title')}
+        </h4>
+        <div className="story-text">
+          <Anchorify text={selectedStory.get('description')}/>
+        </div>
+      </div>
+
+      {
+        userCanCurrentlyEstimate &&
+        <Cards ownEstimate={ownEstimate}/>
       }
-    }
 
-    return (
-      <div className="estimation">
+      {
+        userCanCurrentlyEstimate &&
+        <div className="board-actions">
+          <button type="button" className="pure-button pure-button-primary"
+                  onClick={() => reveal(selectedStory.get('id'))}>
+            {t('revealManually')}
+            <i className="fa fa-hand-paper-o button-icon-right"></i>
+          </button>
+        </div>
+      }
 
-        <div className="selected-story">
-          <h4>
-            {selectedStory.get('title')}
-          </h4>
-          <div className="story-text">
-            <Anchorify text={selectedStory.get('description')}/>
-          </div>
+      {
+        revealed &&
+        <div className="board-actions">
+          <button type="button" className="pure-button pure-button-primary"
+                  onClick={() => newEstimationRound(selectedStory.get('id'))}>
+            {t('newRound')}
+            <i className="fa fa-undo  button-icon-right"></i>
+          </button>
         </div>
+      }
+      {
+        revealed && storyPointConsentAchieved && <PopUp messageType="success" message={t('spConsent')}/>
+      }
+    </div>
+  );
 
-        {
-          userCanCurrentlyEstimate &&
-          <Cards ownEstimate={ownEstimate}/>
-        }
-
-        {
-          userCanCurrentlyEstimate &&
-          <div className="board-actions">
-            <button type="button" className="pure-button pure-button-primary"
-                    onClick={() => reveal(selectedStory.get('id'))}>
-              {t('revealManually')}
-              <i className="fa fa-hand-paper-o button-icon-right"></i>
-            </button>
-          </div>
-        }
-
-        {
-          revealed &&
-          <div className="board-actions">
-            <button type="button" className="pure-button pure-button-primary"
-                    onClick={() => newEstimationRound(selectedStory.get('id'))}>
-              {t('newRound')}
-              <i className="fa fa-undo  button-icon-right"></i>
-            </button>
-          </div>
-        }
-        {
-          revealed && storyPointConsentAchieved && <PopUp messageType="success" message={t('spConsent')}/>
-        }
-      </div>
-    );
+  /**
+   * Tests if the consent was achieved
+   * @param estimations the estimations
+   * @return true if the consent was achieved otherwise false
+   */
+  function isConsentAchieved(estimations) {
```

Comment:
what about

```
 /**
   * Tests if the consent was achieved
   * @param estimations the estimations
   * @return {boolean} true if the consent was achieved otherwise false
   */
  function isConsentAchieved(estimations) {
    // This is the case when no one gave an estimation
    if (estimations.size === 0) {
      return false;
    }

    const estimationValues = estimations.valueSeq();
    var firstValue = estimationValues.get(0);
    return estimationValues.every((v, i) => v === firstValue);
  }
```